README.md

```diff
@@ -845,31 +845,31 @@ U              ungreedy: swap meaning of x* and x*?, x+ and x+?, etc (default fa
 For example the following regular expression would match any document with a field `type` and its value `rest-layer`.
 
 ```json
```

Comment:
Try js or JavaScript in stead of json to get rid of highlight errors when the key is not quoted. This is basically JS styled syntax.